src/test/java/pages/SmartphonePage.java

```diff
@@ -0,0 +1,66 @@
+package pages;
+
+import elements.Checkbox;
+import elements.Text;
+import endpoints.OnlinerEndpoints;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import util.FindElementUtils;
+import util.ScrollUtils;
+import util.WaitUtils;
+
+public class SmartphonePage extends BasePage {
+
+    protected Checkbox cBoxItem;
```

Comment:
there is no point in creating global cBoxItem which you cannot initialize with PF, you are risking to face the issue which we are trying to solve with PF. In such situations just use driver.findElement in the places where you need this element without global variables

Reply:
fixed

Reply:
fixed